### PR TITLE
refresh lightbox (bug 1129375)

### PR DESCRIPTION
**Pull request by spasovski**

### CODE REVIEW

File: src/media/css/lightbox.styl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'lib';
 
 #lightbox {
-    background: rgba(0,0,0,.75);
+    background: rgba($greyscale-light-grey,.9);
```
Comment (ngokevin) [inline src/media/css/lightbox.styl]: space. and other modals are using rgba($greyscale-grey, .8); so let's use that

Comment (spasovski) [inline src/media/css/lightbox.styl]: phil commented that .8 is too transparent last time, agree with keeping things consistent though

Comment (ngokevin) [inline src/media/css/lightbox.styl]: yeah but for greyscale-grey instead of light-grey should make it better

**Comment by ngokevin** [comment]
There's some weird animation with the arrow when I click on it.

**Comment by ngokevin** [comment]
Once you get to the end, the arrows turn grey, but when clicking them they close the lightbox.

**Comment by ngokevin** [comment]
You know what, we can actually test the lightbox. I'd set up a helpers.desktopTest (or use the existing previews test), and have it click the left/right arrows and the disabled arrows, make sure everything works.

**Comment by ngokevin** [comment]
Can you get rid of the thick white border on the close button?

**Comment by ngokevin** [comment]
r+wc

I'm fine with merging this PR, closing the bug, and writing tests right afterwards too.

**Pull request merged**